### PR TITLE
feat: [CCM-8976]: perspective filters fixes

### PR DESCRIPTION
**Pull request by jyotiarora-harness**
### Summary

Perspective filter fixes in case of Null and Not Null operator selection

#### Screenshots
Screenshot 2022-09-13 at 6 21 04 PM

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 #### 1 Issues 
* Code issue 1 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/11210)

**Comment by jyotiarora-harness** [comment]
trigger cypress

**Comment by jyotiarora-harness** [comment]
trigger cypress

File: src/modules/75-ce/components/PerspectiveFilters/FilterPill.tsx [hunk 1]
```diff
@@ -98,6 +98,17 @@ const FilterPill: React.FC<FilterPillProps> = ({ fieldValuesList, removePill, id
 
   const filteredFieldValuesList = fieldValuesList.filter(fieldValue => fieldValue) as QlceViewFieldIdentifierData[]
 
+  const renderOperator = React.useMemo(() => {
```
Comment (rishik30) [inline src/modules/75-ce/components/PerspectiveFilters/FilterPill.tsx:101]: Naming convention could have been better. `renderOperator` is more apt for a function.

**Review by rishik30 — APPROVED** [review]

**Review by jenil-harness — APPROVED** [review]

**Pull request merged**